src/lib/apiTypesV2/Entity.cpp

```diff
@@ -256,21 +256,6 @@ std::string Entity::check(ApiVersion apiVersion, RequestType requestType)
 
 
 
-/* ****************************************************************************
-*
-* Entity::present - 
-*/
-void Entity::present(const std::string& indent)
-{
-  LM_T(LmtPresent, ("%sid:        %s", indent.c_str(), id.c_str()));
```

Comment:
After the removals in this PR, is the `LmtPresent` trace being used in some place? Otherwise, it should removed from the tracelevel.h file where it is defined.

Reply:
It should most probably be removed.
`LmtPresent` was invented for this purpose

Reply:
Hi @fgalan Sir and @kzangeli Sir

There are some methods in which  LmtPresent is used as  you informed in below comment.
eg:
/* ****************************************************************************
*
* jsonUncrPresent - 
*/
void jsonUncrPresent(ParseData* parseDataP)
{
  if (!lmTraceIsSet(LmtPresent))
    return;

  LM_T(LmtPresent,("\n\n"));

  parseDataP->uncr.res.subscriptionId.present("");
}

Thanks

Reply:
...althought, it seems this PR also fixes the methods mentioned in #3206. Is that correct? That would be great, to fix both issues in the same PR :)

(...and, if at the end there is no usage of LmtPresent, then remove it also)

Reply:
Hi @fgalan Sir,

Yes you are right. Through this PR we can fix both the bugs (#3206 and #3199) .

Thanks